neuraxle/steps/loop.py

```diff
@@ -146,6 +161,36 @@ def should_resume(self, data_container: DataContainer, context: ExecutionContext
         return False
 
 
+class ContinueException(Exception):
+    pass
+
+
+class BreakException(Exception):
```

Comment:
The `KeyboardInterrupt` exists. This suggests possibly naming your new exception a `BreakInterrupt` instead of a `BreakException`. Same for the ContinueInterrupt.